docs/interaction_based_testing.adoc

```diff
@@ -1030,7 +1030,84 @@ a subscriber named Barney instead.
 [MockingStaticMethods]
 === Mocking Static Methods
 
-(Think twice before using this feature. It might be better to change the design of the code under specification.)
+Spock supports two ways to mock static methods:
+
+* `Mock/Stub/SpyStatic` static mocks: Works with Java and Groovy, but requires a mock maker supporting this, e.g. `mockito`.
```

Comment:
Should this link to the section about mock makers?

Reply:
I have added a link to the extensions.